pkg/infra/log/log.go

```diff
@@ -433,9 +442,55 @@ func ReadLoggingConfig(modes []string, logsPath string, cfg *ini.File) error {
 		configLoggers = append(configLoggers, handler)
 	}
 
+	var err error
+	root.gokitLogActivated, err = isNewLoggerActivated(cfg)
+	if err != nil {
+		return err
+	}
 	if len(configLoggers) > 0 {
 		root.initialize(configLoggers)
 	}
 
 	return nil
 }
+
+// This would be removed eventually, no need to make a fancy design.
+// For the sake of important cycle I just copied the function
+func isNewLoggerActivated(cfg *ini.File) (bool, error) {
+	section := cfg.Section("feature_toggles")
+	toggles, err := readFeatureTogglesFromInitFile(section)
+	if err != nil {
+		return false, err
+	}
+	return toggles["gokitlogActivated"], nil
```

Comment:
Suggestion for alternative toggle:

```suggestion
	return toggles["newlog"], nil
```

Not sure the internal lib matters, and "activated" is redundant to the toggle being set 🙂